docs/modeling/triggers.md

```diff
@@ -0,0 +1,113 @@
+| `Description` | A free text description of what the trigger does | A trigger to start a process when an approval email is received. | 
+
+### Events
+Events are created when the configured event criteria are met. 
+The properties for trigger events are: 
+
+| Property | Description | Example | 
+| -------- | ----------- | ------- | 
+| `Connector` | The [connector](../modeling/connectors/README.md) instance to use for an event | email-connector-1 | 
+| `Event` | The [connector event](../modeling/connectors/README.md#events) to use for this trigger | EMAIL_RECEIVED | 
+| `Input values` | The inputs for the trigger event. The properties are set in the [connector event](../modeling/connectors/README.md#events) definition | | 
+| `source` | The event source is automatically set and is the channel that the event is sent to | connector.email-connector-1.EMAIL_RECEIVED | 
+
+Events are stored as a `source` containing the connector and event to use and a set of `inputs` containing the values for the trigger event, for example using the `EMAIL_RECEIVED` event for the [email connector](../modeling/connectors/ootb/email.md):
+
+```json
+"event": {
+	"inputs": {
+		"pattern": "",
+		"echo": "Email pattern matched. Starting a process instance.",
+		"echoError": "We hit a problem starting a process instance for that message. "
+	},
+	"source": "connector.email-connector-1.EMAIL_RECEIVED"
+}
+```
+
+The [email connector](../modeling/connectors/ootb/email.md), [Slack connector](../modeling/connectors/ootb/slack.md) and [Twilio connector](../modeling/connectors/ootb/twilio.md) have events defined that can be used for triggers.
+
+### Actions
+Actions are the trigger payload that is created and sent when an event is created. The actions that can be configured are:
+
+* [Start a process](#start-a-process)
+* [Send a signal](#send-a-signal)
+
+#### Start a process
+The action to start a process will begin a specific process instance when event are met. The payload for the start a process action is:
+
+| Property | Description | Example | Required | 
+| -------- | ----------- | ------- | -------- |
+| `processDefinitionKey` | The ID of the process to start | Process_YjqRw3n2 | Yes |
+| `name` | The name of the process instance | approval-process | Yes | 
+| `businessKey` | An optional business key ID for the process instance | 14240 | No | 
+| `variables` | Values from the trigger can be mapped to [process variables](../modeling/processes/variables.md). For a connector this will include the output parameters configured in the [connector event](../modeling/connectors/README.md#events) definition. | | No | 
+| `source` | The source is automatically set and is the channel that the action is sent to | command-consumer | 
+
+Actions are stored as a `payload`, for example using the start a process action for the `EMAIL_RECEIVED` event of the [email connector](../modeling/connectors/ootb/email.md):
```

Comment:
I think it is worth to mention (I don't know where should be the best place) that the action's parameters that form the payload references mainly to the event output parameters (just a suggestion)

Reply:
Added a line about this for triggers and connector events.